src/slack_block_util.ts

```diff
@@ -333,6 +359,233 @@ export function getErrorSlackView(
   };
 }
 
+interface OpenCloseModalProps {
```

Comment:
Thoughts on moving all the open/close channel logic from slack_block_util to slack_block_entrypoint_util or something like that? Since it's ~doubling the size of the file.